packages/frontend2/src/app/utils/getMetadata.ts

```diff
@@ -0,0 +1,27 @@
+import { type Metadata } from 'next'
+
+interface MetadataParams extends Metadata {
+  title: string
+  image: string
+  description?: string
+}
+
+export function getMetadata(params: MetadataParams): Metadata {
```

Comment:
Metadata is automatically merged in the app router, there's no need for this kind of stuff.

Reply:
oh, forgot about that